.ci/scripts/dart_unit_tests_win32.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Exclude flutter_image because its tests need a test server, so are run via custom_package_tests.
+# Exclude xgd_directories because it is a Linux only package.
+dart pub global run flutter_plugin_tools test --exclude=flutter_image,fuchsia_ctl,xgd_directories \
```

Comment:
fuchsia failure
```
00:00 +8 -1: test\emulator_test.dart: Emulator prepare environment runs as expected [E]
  Expected: true
    Actual: <false>
  
  package:test_api              expect
  test\emulator_test.dart 56:7  main.<fn>.<fn>
  
00:00 +9 -1: test\image_paver_test.dart: Tar fails
```
need advice on how or whether we should fix it instead of skipping

Reply:
I would file an issue with a TODO link, so that it doesn't block enabling other tests, and we can route the issue to someone who knows whether the exclusion should be permanent.